modules/builds-source-secret-ssh-key-auth.adoc

```diff
@@ -6,35 +6,30 @@
 // This module can be included from assemblies using the following include statement:
 // include::<path>/builds-source-secret-ssh-key-auth.adoc[leveloffset=+1]
 
-// Base the file name and the ID on the module title. For example:
-// * file name: doing-procedure-a.adoc
-// * ID: [id='doing-procedure-a']
-// * Title: = Doing procedure A
-
 [id='source-secrets-ssh-key-auth-{context}']
 = Source code SSH key authentication
 
 The repository keys are usually located in the *_$HOME/.ssh/_* directory, and
 are named `id_dsa.pub`, `id_ecdsa.pub`, `id_ed25519.pub`, or `id_rsa.pub` by
 default.
 
-. Prerequisites
+.Prerequisites
 
 * SSH key based authentication requires a private SSH key.
 
 .Procedure
 
+Two files are created: the public key and a corresponding private key (one of
```

Comment:
I think there's another step hidden in here.

Reply:
Thanks! I'll update in just a bit :smiley_cat: